docs/concepts/trust_levels.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 🎚 Trust Levels
+description: What level of "community trust" does SourceCred depend on
+---
+
+SourceCred is intended to be used by communities to divy up rewards, in the
+form of cred and grain. As such, SourceCred needs to be resilient to gaming by
+people who explicitly (but perhaps covertly) attempt to maximize their rewards
+at all other costs.
+
+The kind of gaming that will happen depends on the ambient level of trust and
+social cohesion within the community. A small community of like-minded and
+highly aligned people might see very little gaming, with most people explicitly
+working together to make sure everyone is fairly credited. On the other hand, a
+larger and more anonymous community may have many selfish actors who just want
+to increase their scores without regard for the broader mission.
+
+We've developed the conept of "Trust Levels" to try to categorize communities
```

Comment:
Who is we? If we want to make the SC team more _personable_, we could say something like, "the SourceCred team has developed..." If we want SC to seem more _powerful_ in the way of seeming large, we could say, "SourceCred has developed..."